proposed/form.md

```diff
@@ -0,0 +1,101 @@
+# Parsy Pattern Design Doc
+
+Parsy
+
+#Form
+
+* Author: `Lorenzo Wallerlei, lorenzo@polderknowledge.nl`
+* Date: `19-01-2017`
+* Repository name: `FormComponent`
+
+## Goals
+
+This document is the proposal for the form component, it describes what should go in a form and how a form should react 
+to user input.
+
+## References
+
+http://getbootstrap.com/components/#input-groups
+
+http://foundation.zurb.com/sites/docs/forms.html
+
+http://webaim.org/techniques/forms/controls
+
+## Requirements
+
+- All the inputs within the form SHOULD contain a tab index.
+- All the inputs SHOULD have error messages if there is some sort of validation on that specific input
+- Inputs MUST have labels
+- Form MUST have an action attribute
+- If there is a collection of inputs it MUST be surrounded by a fieldset.
+- If you use an fieldset you MUST use a legend for it.
+
+# Case studies `<Optional>`
+
+Not applied.
+
+## Existing implementations
+
+[https://www.tax.service.gov.uk/gg/sign-in?continue=%2Faccount&origin=unknown]
+
+What it does good.
+- Form fields are described by labels.
+- Flow of the tab indexes is natural.
+- Clear error messages.
+- Submits with the enter key.
+- Has a form action so it could be submitted without the use of javascript.
+
+What it could do better.
+- Refocus after validation.
+
+## Proof of concept
+
+This pattern has no proof of concept. It is a set of rules to follow for making a good structured, valid and accessible
+form.
+
+## API
+
+Not used.
+
+## Constraints
+
+TBD.
+
+## Classes
+
+    Name       | Type     | Description
+    -----------|----------|-------------
+
+    has-errors | "global" | Can make it easier for other componts to know if the form has errors.
+
+
+## Accessibility `<Important>`
+
+TBD.
+
+### How to use the component with a keyboard only
+
+This MUST be entirely usable with only the tab key.
```

Comment:
How would you submit it with a tab key? You need to clarify this sentence.You probably mean to say that it should be usuable with the keyboard.